### PR TITLE
Improved manage admin UI. Added Email Validation, delete confirm pop-up

### PR DESCRIPTION
**Pull request by adityab17190**
Addressed User Story #127- https://tree.taiga.io/project/yousuf_h-human-agent-interaction/us/127.
 Following things are done-

1) Show delete confirm pop-up when super admin deleted an admin
2) Add Email Validation on UI
3) Add Email Validation on backend to allow only asu.edu emails.
4) Fixed Issue #25
5) Also prevented super admin from being shown in list of admins for deletion

### CODE REVIEW

File: MainApp/public/app/controllers/authCtrl.js [hunk 1]
```diff
@@ -82,7 +83,17 @@ angular.module('authControllers', ['authServices'])
             });
         };
 
-
+        let getClientId=function(){
+            var metaTags=document.getElementsByTagName('meta');
```
Comment (yousuf031991) [inline MainApp/public/app/controllers/authCtrl.js:87]: Does this fix [the meta tag issue](https://github.com/yousuf031991/HumanAgentInteraction/issues/25) ?

Comment (adityab17190) [reply MainApp/public/app/controllers/authCtrl.js:87]: @yousuf031991  Yes.

File: MainApp/public/app/views/pages/admin/manageAdmin.html [hunk 2]
```diff
@@ -10,7 +10,7 @@ <h3 class="panel-title">Add New Admin</h3>
             <div class="panel-body">
                 <form ng-submit="createNewAdmin();">
                     <label>Email Id:</label>
-                    <input class="form-control" type="text" name="emailId" placeholder="Enter email"
+                    <input class="form-control" type="email" name="emailId" placeholder="Enter email"
```
Comment (super9user) [inline MainApp/public/app/views/pages/admin/manageAdmin.html:13]: Is there an error displayed to user if email is not @asu.edu email?

Comment (adityab17190) [reply MainApp/public/app/views/pages/admin/manageAdmin.html:13]: @super9user Yes. Check api.js- line no. 145

Comment (super9user) [reply MainApp/public/app/views/pages/admin/manageAdmin.html:13]: Cool.
Looks good to me +1

**Review by super9user — APPROVED** [review]

**Review by yousuf031991 — APPROVED** [review]
+1

**Pull request merged**